ocs_ci/ocs/ui/views.py

```diff
@@ -1989,7 +1989,61 @@
     "alerting_rule_details_link": ("//a[normalize-space()='{}']", By.XPATH),
 }
 
-
+bucket_tab = {
+    "create_bucket_button_obc": (
+        "//*[@id='content-scrollable']/section/div[2]/div[1]/div/div[2]/div[1]/div[2]",
+        By.XPATH,
+    ),
+    "storage_class_dropdown": (
+        "[data-test='sc-dropdown']",
+        By.CSS_SELECTOR,
+    ),
+    "storage_class_noobaa_option": (
+        "div.pf-v5-c-dropdown div:nth-of-type(2)",
+        By.CSS_SELECTOR,
+    ),
+    "create_bucket_button_s3": (
+        "div:nth-of-type(2) > div.pf-v5-c-tile__body",
+        By.CSS_SELECTOR,
+    ),
+    "bucket_name_input": (
+        "[data-test='bucket-name']",
+        By.CSS_SELECTOR,
+    ),
+    "submit_button_obc": (
+        "[data-test='obc-create']",
+        By.CSS_SELECTOR,
+    ),
```

Comment:
apart from these locators: [data-test='bucket-name'], [data-test='obc-create'], //*[@id='yaml-create'], //button[contains(text(), 'Upload')]" that are unique other locators cant be considered good. HTML changes and even slight change of the DOM will shift indexes, relation between css elements, etc.

We need to find unique locators, not using indexes of common elements. Also this relation with multiple parents and childen elements`]/section/div[2]/div[1]/div/div[2]/div[1]/div[2]` may change even without changes from devs, but with different user inputs. 

In opposite, rigid locators, like "//*[@id='yaml-create'] or [data-test='bucket-name'] will continue work even with changes on page from development.